lib/modules/manager/buildpacks/extract.spec.ts

```diff
@@ -103,13 +103,15 @@ id = "not/valid"`,
           datasource: 'buildpacks-registry',
           currentValue: '1.0.0',
           packageName: 'example/foo',
+          depName: 'example/foo',
```

Comment:
Duplicating packageName into depName is undesirable. Is this change avoidable?

Reply:
We switched to use depName now.

Reply:
Why switch? If you have only one, then packageName is preferred.